v2/app/collegevine-hub/CVHubViewer.jsx

```diff
@@ -1509,6 +1517,11 @@ var PivotViewer = (Pivot.PivotViewer = function (
                 ) {
                   hoveredItem = item
                   hoveredItemIndex = i
+                  anyItemHovered = true
```

Comment:
Could we reuse `hoveredItem` for this?

Reply:
I think I should be able to, I'll try it

Reply:
@gasi we actually can't reuse `hoveredItem` for this, I tested it and realized that it will fire an `itemFocus` event and then an `itemBlur` event.  I will look into exactly why that is again but for the time being I think it is necessary to keep this logic in place.